.circleci/config.yml

```diff
@@ -5,28 +5,46 @@ orbs:
   aws-cli: circleci/aws-cli@0.1.19
   k8s: circleci/kubernetes@0.11.0
 
-jobs:
-  build:
+references:
```

Comment:
I actually realized what was wrong with this this morning, when I noticed CircleCI was waiting for a "build" job to report a successful status even though there was no build job defined any more.

Then I remembered CircleCI requires a job named "build" when you do not use workflows. I'm wondering if there's something messed up by not having a job *or* a workflow called "build".

Reply:
If we can't change that name in CircleCI, we could just rename the `build_x86_64` to `build` again, since that would be the default.

Reply:
> If we can't change that name in CircleCI, we could just rename the `build_x86_64` to `build` again, since that would be the default.

I'm going to see if this issue just goes away. I think maybe (?) the old config YAML was cached somehow and that was the reason for the empty "build" step status check being expected...